templates/home.html

```diff
@@ -40,4 +41,7 @@ <h1>Paricia</h1>
                 </tbody>
         </table>
 </div>
+<div class="{% plotly_class name='StationsMap' %} card" style="height: 100%; width: 100%">
+{% plotly_app name="StationsMap" initial_arguments=django_context ratio=1 %}
+</div>
```

Comment:
This is WAY more interesting for most users than the API specifications. Would move it to the top of the landing page.